docs/content/latest/explore/ysql-language-features/advanced-features/partitions.md

```diff
@@ -99,7 +99,39 @@ To create a new partition that contains only the rows that don't match the speci
 CREATE TABLE order_changes_default PARTITION OF order_changes DEFAULT;
 ```
 
-Optionally, you can create an index on the key columns and other indexes for every partition, as follows:
+Optionally, you can create indexes on a partitioned table.
+```sql
+yugabyte=# CREATE INDEX ON order_changes (change_date);
+```
+
+This automatically creates indexes on each partition.
```

Comment:
Suggestion:

This automatically creates indexes on each partition, as demonstrated by the following output: